### ISSUE TITLE
Occurrence pages denied by robots.txt

### ISSUE THREAD

**Issue by dshorthouse** [gbif/portal-feedback#3734]
The https://gbif.org/robots.txt directives evidently deny indexing of occurrence pages and yet there are `og:title` headers, which suggests you do in fact wish for occurrence pages to be indexed. The twitter card validator for occurrence pages says:

```
ERROR: Fetching the page failed because it's denied by robots.txt.
```

...which severely limits their reach on social networks.

**Comment by MortenHofft** [comment]
Hi David
We deny robots as we have had issues with Google in the past if we allow them to index 2 billion occurrences (they classified us as spam I believe)

But we add open graph tags nonetheless - if I remember correctly we generate them automatically. But I do not see any reason for explicitly removing them. Facebook reads the pages just fine. At least their preview tool works fine with occurrence pages.
Perhaps we have to choose between appearing on Twitter and Google then.

**Comment by dshorthouse** [comment]
Really?! That's unfortunate. I was hopeful that there'd by a Twitter card with media to go along with tweets like this https://twitter.com/bionomiatrack/status/1458842194689728525?s=21 but apparently Twitter looks at robots.txt and then shows nothing. iNaturalist seems not to have taken the same view.

**Comment by MortenHofft** [comment]
Yes it is a shame. Perhaps there is a way to satisfy both, but I do not know how to I must admit. If you or anyone else do, then please let me know. Then I would be happy to adjust the site tags and robots file.

My best guess for an alternative approach, would be to allow indexing of individual occurrences, but not following links. 
And to make sure that they do not get indexed to begin with, by disallowing all pages that link to them (e.g. occurrence search). That way only individual pages that have links from elsewhere would be picked up. But I *think* that might have been attempted in the past without success. We can revisit it, Google might also behave different now than 5 years ago.

**Comment by dshorthouse** [comment]
A `nofollow` directive, absent the `noindex`, might be the more logical approach here, but I have no idea what this does to SEO with respect to internal linking. I do agree that a sitemap that contains **ALL** the occurrence pages would not likely be a good approach.

**Comment by MattBlissett** [comment]
We have `User-Agent: *` at the start, when we could perhaps be less restrictive and `Allow: ` Twitter, or only disallow some known search engines.
